mantle/platform/qemu.go

```diff
@@ -1394,9 +1394,20 @@ func (builder *QemuBuilder) Exec() (*QemuInstance, error) {
 	}
 	argv = append(argv, "-smp", fmt.Sprintf("%d", builder.Processors))
 
+	if builder.Firmware == "" {
```

Comment:
Why are we lowering this into the QEMU code, rather than keeping it in `syncOptionsImpl`, which is intended to clean up user-provided options?

Reply:
I see `qemu.go` as lower level than kola - longer term I think we want to break it out as its own Go module.

I also think we should be offering our qemu bits at least as a user-facing CLI tool - xref https://github.com/coreos/fedora-coreos-tracker/issues/235